android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt

```diff
@@ -311,13 +306,31 @@ constructor(
   }
 
   suspend fun onQuestionnaireSubmission(questionnaireSubmission: QuestionnaireSubmission) {
+    // Patient/1423423423
+    questionnaireSubmission.questionnaireResponse.subject?.run {
+      val resource: Resource =
+          defaultRepository.loadResource(questionnaireSubmission.questionnaireResponse.subject)
+      defaultRepository.addOrUpdate(resource = resource)
+    }
+    // update group
```

Comment:
Are we still using the `groupResoruce` even after deciding to use the `questionnaire.action.params`?